apollo-client/src/main/java/com/ctrip/framework/apollo/monitor/api/ApolloClientBootstrapArgsMonitorApi.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2022  Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.monitor.api;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * @author Rawven
+ */
+public interface ApolloClientBootstrapArgsMonitorApi {
+
+  /**
+   * get startup params by key
+   */
+  default String getStartupArg(String key) {
+    return getBootstrapArgs().get(key);
+  }
+
+  /**
+   * get bootstrap args map
+   */
+  default Map<String, String> getBootstrapArgs() {
+    return Collections.emptyMap();
+  }
+
+  /**
+   * config service url
+   */
+  default String getConfigServiceUrl() {
+    return "";
```

Comment:
All method can reuse getBootstrapArgs

```suggestion
    return getBootstrapArgs().getOrDefault(XxxKey, "");
```

Reply:
nice idea